src/recordlinker/linkage/utils.py

```diff
@@ -73,7 +70,7 @@ def datetime_to_str(
     """
     # Handle None or empty string
     if input_date is None or input_date == "":
-        return input_date
+        return ""
```

Comment:
What was the reason for this change? (Do we want to make sure two empty strings are evaluated as equal?)

Reply:
There was a potential for a bug here.  The signature of the method states a `str` should be returned, however if the `input_date` is None, `None` could be returned.  I changed it to guarantee that `datetime_to_str` always returns a `str`.